rootfs_linux.go

```diff
@@ -62,6 +69,21 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 	return nil
 }
 
+func mountCmd(cmd string) error {
+	if cmd != "" {
+		args := strings.Split(cmd, " ")
+		mCmd := exec.Command(args[0], args[1:]...)
+		if err := mCmd.Start(); err != nil {
```

Comment:
@rhatdan we can use mCmd.Run() here instead of {Start, Wait} commands.
http://golang.org/pkg/os/exec/#Cmd.Run